x/pairing/keeper/badge_used_cu.go

```diff
@@ -0,0 +1,74 @@
+package keeper
+
+import (
+	"github.com/cosmos/cosmos-sdk/store/prefix"
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lavanet/lava/x/pairing/types"
+)
+
+// SetBadgeUsedCu set a specific badgeUsedCu in the store from its index
+func (k Keeper) SetBadgeUsedCu(ctx sdk.Context, badgeUsedCu types.BadgeUsedCu) {
+	store := prefix.NewStore(ctx.KVStore(k.storeKey), types.KeyPrefix(types.BadgeUsedCuKeyPrefix))
+	b := k.cdc.MustMarshal(&badgeUsedCu)
+	store.Set(types.BadgeUsedCuKey(
```

Comment:
`badgeUsedCu.BadgeUsedCuMapKey` is already bytes.
I assume you add '/' suffix to prevent hypothetical prefix issues (one badge key is somehow prefix of another) right?
anyway, why not create the key with a trailing '/', so you can then use it as is and not need to repeatedly add the slash for all store.XXXXX() calls?

Reply:
this is ignite generated code